src/main/java/AlexLink/HomeWork/Exception/StartException.java

```diff
@@ -0,0 +1,10 @@
+package AlexLink.HomeWork.Exception;
+
+public class StartException extends Exception {
```

Comment:
@svetamolchanova Is this expected that this class extends Exception, not RuntimeException?

Reply:
fixed